fe2-android/app/src/main/java/com/github/dedis/popstellar/utility/handler/data/LaoHandler.java

```diff
@@ -159,15 +161,14 @@ public void handleStateLao(HandlerContext context, StateLao stateLao)
     // TODO: verify if lao/state_lao is consistent with the lao/update message
 
     Lao lao = laoView.createLaoCopy();
-
     lao.setId(stateLao.getId());
     lao.setWitnesses(stateLao.getWitnesses());
     lao.setName(stateLao.getName());
     lao.setLastModified(stateLao.getLastModified());
     lao.setModificationId(stateLao.getModificationId());
 
     PublicKey publicKey = keyManager.getMainPublicKey();
-    if (laoView.isOrganizer(publicKey) || laoView.isWitness(publicKey)) {
+    if (lao.getOrganizer().equals(publicKey) || lao.getWitnesses().contains(publicKey)) {
```

Comment:
Why did you change the previous condition `laoView.isOrganizer(publicKey) || laoView.isWitness(publicKey)`, isn't it better to use such methods so that it's cleaner?

Reply:
I did this to make it consistent with the same line in `handleCreateLao` (line 67), but you are right, the other code is cleaner. I will revert this and instead change the line in `handleCreateLao` to use the `LaoView` too.